package.json

```diff
@@ -37,23 +37,26 @@
     "debuglog": "^1.0.1",
     "js-yaml": "^3.2.6",
     "path-to-regexp": "^1.2.1",
-    "swaggerize-routes": "^1.0.0"
+    "swaggerize-routes": "^2.0.0-alpha.1"
```

Comment:
will we wait for the official release of swaggerize-routes 2? 
Would be helpful to link to the PR for that also, for the benefit of other folks reading this PR

Reply:
Good idea, I just have added the link in the PR description. Thanks